deprecate.dd

```diff
@@ -9,47 +9,45 @@ $(SPEC_S Deprecated Features,
+        $(TROW $(DEPLINK Base Class Protection),                                  2.058, 2.058,  2.067,  2.072 )
+        $(TROW $(DEPLINK Windows 3.x and Windows 9x support),                     2.058, N/A,    N/A,    2.058 )
+        $(TROW $(DEPLINK typedef),                                                2.057, 2.057,  2.067,  2.072 )
+        $(TROW $(DEPLINK Using * to dereference arrays),                          ?,     2.057,  2.067,  (never) )
+        $(TROW $(DEPLINK invariant as an alias for immutable),                    2.057, 2.057,  2.064,  2.066 )
+        $(TROW $(DEPLINK Non-final switch statements without a default case),     2.054, 2.054,  2.068,  (never) )
+        $(TROW $(DEPLINK Hiding base class functions),                            2.054, 2.054,  2.068,  (never) )
+        $(TROW $(DEPLINK Octal literals),                                         2.054, 2.053,  2.067,  (never) )
+        $(TROW $(DEPLINK C-style function pointers),                              ?,     2.050,  2.067,  (never) )
+        $(TROW $(DEPLINK Using length in index expressions),                      ?,     2.041,  &nbsp;, 2.061 )
+        $(TROW $(DEPLINK Escape string literals),                                 ?,     2.026,  2.061,  2.067 )
+        $(TROW $(DEPLINK volatile),                                               2.013, 2.013,  2.067,  2.072)
+        $(TROW $(DEPLINK HTML source files),                                      ?,     2.013,  N/A,    2.061 )
+        $(TROW $(DEPLINK Overriding without override),                            ?,     2.004,  2.072,  (never) )
+        $(TROW $(DEPLINK Lower case 'l' suffix for integer literals),             ?,     1.054,  0.174,  (never) )
+        $(TROW $(DEPLINK Variable shadowing inside functions),                    ?,     0.161,  &nbsp;, 2.061 )
+        $(TROW $(DEPLINK Upper case 'I' suffix for imaginary literals),           ?,     0.154,  2.061,  (never) )
+        $(TROW $(DEPLINK if (v; e)),                                              ?,     0.149,  2.061,  2.068 )
+        $(TROW $(DEPLINK Removing an item from an associative array with delete), ?,     0.127,  2.061,  2.067 )
+        $(TROW $(DEPLINK .offset property),                                       ?,     0.107,  2.061,  2.067 )
+        $(TROW $(DEPLINK .size property),                                         ?,     0.107,  0.107,  2.061 )
+        $(TROW $(DEPLINK .typeinfo property),                                     ?,     0.093,  2.061,  2.067 )
     )
 
     $(DL
         $(DT Spec)
         $(DD Removal from the Specification)
-        $(DT Warn)
-        $(DD The compiler emits a warning if the feature is used)
         $(DT Dep)
-        $(DD The compiler issues an error if not compiled with the -d switch)
+        $(DD The compiler warns by default, issues an error with the -de switch, and can be silenced with the -d switch)
```

Comment:
Not sure what the use-case for `-dw` is. If you have a better wording, just make a PR.